### DISCUSSION TITLE
Fuxa communication with multiple Modbus RTU slaves

### DISCUSSION THREAD

**Discussion by robsori** [frangoteam/FUXA#799] [General]
Hi,
Has anyone tried to communicate with Fuxa through multiple devices via Modbus RTU? I know that querying client devices through serial communication must be done one at a time. I did not see a possibility of chained delay when selecting the Modbus RTU communication driver. E.g:
1. Communicating with device 1
Pause=1 sec
2. Communicating with device 2
Pause=1 sec
.
.
n. Communicating with device n
Pause=1 sec
Concurrent serial communication will give rise to communication errors.

**Comment by rvbatista** [comment]
Hello, #211 is about the same.

I'm working on this issue.
